### ISSUE TITLE
google access token to be stored in-memory

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#4196]
In another issue: switch access token to be also stored in-memory instead of storage.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/issues/4194#issuecomment-994852459_

similar/same as how id token is stored in memory

**Comment by tomholub** [comment]
will need to also update additional settings -> experimental -> wipe access token

google_access_token should be removed from local storage model

I don't think we need to actually wipe it from local storage (no need to do migration) since these access tokens are short lived. It will just expire.

**Comment by tomholub** [comment]
This is for after #4194 / #4195 lands.
